hack/update-codegen.sh

```diff
@@ -97,7 +97,7 @@ ${REPO_ROOT_DIR}/hack/update-reference-docs.sh
 group "Generate CRD schemas"
 # TODO: Drop patched version once URL issue is fixed.
 # See: https://github.com/kubernetes-sigs/controller-tools/issues/552.
-run_go_tool github.com/markusthoemmes/controller-tools/cmd/controller-gen controller-gen \
+run_go_tool github.com/markusthoemmes/controller-tools/cmd/controller-gen@knative-specific controller-gen \
```

Comment:
Have we already / are we planning to PR these changes back to main repo?

Reply:
Partially, yes, though I don't quite know yet how we could be doing the podspec related customization upstream. Part of the idea here was to have a fork in knative/knative-sandbox to have these specifica as close to our code as possible. I even toyed pulling the PodSpec override hardcoding into a config file that could then live in Serving. That scheme could potentially be upstreamed but would require quite some discussion.

Reply:
Ah, I should adjust the comment anyway since it's no longer accurate.

Reply:
Yeah, something like the above comment is what I was angling towards. `3` makes it seem like it may be a somewhat-long-running fork though.. if so is it even maybe worth a -sandbox repo to avoid it living in your personal branch or does that feel like overkill?

Reply:
Nope, that's what was the plan anyway

> Part of the idea here was to have a fork in knative/knative-sandbox to have these specifica as close to our code as possible.

Reply:
neat, okie if we have a plan to get off the personal fork seems legit to land it temporarily in the mean time